build.json

```diff
@@ -11,10 +11,14 @@
   "MonoFramework": "framework-5.2.0.213.zip",
   "MonoMSBuildRuntime": "Microsoft.Build.Runtime.Mono-alpha5.zip",
   "MonoMSBuildLib": "Microsoft.Build.Lib.Mono-alpha5.zip",
-  "MainProject": "OmniSharp",
+  "HostProjects": [
+    "OmniSharp.Stdio",
```

Comment:
I'm wondering if we should rename `OmniSharp.Stdio` to `OmniSharp`. Thoughts?

Reply:
We could, but then it obscures that we have different host projects (I don't have a preference)

Reply:
I'm betting folks will figure it out. :smile: